### ISSUE TITLE
Remove Opera from browser support table

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [webcomponents/webcomponents.org#1156]
Chrome, Safari, Firefox and Edge use the different (their own) engines, but since 2013 Opera uses the same engine as Chrome.

What about removing Opera from the browser support table?

**Comment by samuelli** [comment]
Yeah, not opposed to that. Send a PR?

**Comment by FluorescentHallucinogen** [comment]
@samuelli Yep, I'll create the pull request after #1151 and #1152 have been merged. :wink:
